RELEASES.md

```diff
@@ -1,3 +1,118 @@
+Version 1.40.0 (2019-12-19)
+===========================
+
+Language
+--------
+- [You can now use tuple `struct`s and tuple `enum` variant's constructors in
+  `const` contexts.][65188] e.g.
+
+  ```rust
+  pub struct Point(i32, i32);
+
+  const ORIGIN: Point = {
+      let constructor = Point;
+
+      constructor(0, 0)
+  };
+  ```
+
+- [You can now mark an `enum` or `struct` with the `#[non_exhaustive]` attribute to
+  indicate that there may be variants or fields added in the future.][64639]
+  For example this requires adding a wild-card branch (`_ => {}`) to any match
+  statements on an non exhaustive `enum`. [(RFC 2008)]
+- [You can now use function like procedural macros in `extern` blocks and in
+  type positions.][63931] e.g. `type Generated = macro!();`
+- [Function like and attribute procedural macros can now emit
+  `macro_rules!` items, so you can now have your macros generate macros.][64035]
+- [The `meta` pattern matcher in `macro_rules!` now correctly matches the modern
+  attribute syntax.][63674] E.g. `(#[$m:meta])` now matches `#[attr]`,
+  `#[attr{list}]`, `#[attr[list]]`, and `#[attr(list)]`.
+
+Compiler
+--------
+- [Added tier 3 support\* for the
+  `thumbv7neon-unknown-linux-musleabihf` target.][66103]
+- [Added tier 3 support for the
+  `aarch64-unknown-none-softfloat` target.][64589]
+- [Added tier 3 support for the `mips64-unknown-linux-muslabi64`, and
+  `mips64el-unknown-linux-muslabi64` targets.][65843]
+
+* Refer to Rust's [platform support page][forge-platform-support] for more
+  information on Rust's tiered platform support.
+
+Libraries
+---------
+- [The `is_power_of_two` method on numeric types is now a `const` function.][65092]
+
+Stabilized APIs
+---------------
+- [`f32::to_be_bytes`]
+- [`f32::to_le_bytes`]
+- [`f32::to_ne_bytes`]
+- [`f64::to_be_bytes`]
+- [`f64::to_le_bytes`]
+- [`f64::to_ne_bytes`]
+- [`Option::flatten`]
+- [`Option::as_deref`]
+- [`Option::as_deref_mut`]
+- [`HashMap::get_key_value`]
+- [`BTreeMap::get_key_value`]
+- [`slice::repeat`]
+- [`mem::take`]
+- [`UdpSocket::peer_addr`]
+
+Cargo
+-----
+- [Cargo will now always display warnings, rather than only on
+  fresh builds.][cargo/7450]
+- [The `overrides` key in `profiles` section of `Cargo.toml` has been renamed
```

Comment:
`overrides` is still a nightly-only feature.  AIUI, the releases usually don't include nightly-only changes?

Reply:
Yes, I misread the PR and didn't see this behind the unstable flag.